web/router.ex

```diff
@@ -50,6 +52,8 @@ defmodule CodeCorps.Router do
     resources "/user-skills", UserSkillController, except: [:update]
     resources "/role-skills", RoleSkillController, except: [:update]
 
+    resources "/organization-memberships", OrganizationMembershipController, only: [:index, :show, :create, :update, :delete]
```

Comment:
I can do that, but the slug stuff needs to be on the bottom, so I'll keep it there regardless of ordering.